dev/conformance/runner.ts

```diff
@@ -514,17 +521,91 @@ function runTest(spec: ConformanceProto) {
   );
 }
 
+/**
+ * Parses through the TestFile and changes all instances of Timestamps encoded
+ * as strings to a proper protobuf type since protobufJS does not support it at
+ * the moment.
+ */
+function updateTimeStamp(obj: {[key: string]: {}}) {
+  const fieldNames = ['updateTime', 'createTime', 'readTime'];
+  for (const key of Object.keys(obj)) {
+    if (fieldNames.includes(key) && typeof obj[key] === 'string') {
+      obj[key] = timestamp((obj[key] as unknown) as string);
+    } else if (typeof obj[key] === 'object') {
+      updateTimeStamp(obj[key]);
+    }
+  }
+}
+
+/**
+ * Convert a string TimeStamp, e.g. "1970-01-01T00:00:42Z", to its protobuf
+ * type.
+ */
+function timestamp(text: string): {[key: string]: number} {
```

Comment:
convertTimestamp()?

Come to think of it, if you don't think this is going to be temporary, can you explore adding it to `convertInput` which already exists in the conformance tests?

Reply:
We need to convert the timestamp before creating the `TestFile` with `fromObject`, so adding it into `convertInput` would be too late. However, I could pull both of the functions into a `normalizeInput` instead of having `normalizeTimestamp` and `normalizeInt32Value`. Thoughts?

Reply:
Let's leave it is as is, but rename this function to `convertTimestamp`. I would expect a `timestamp` function to take seconds and nanos and return a Timestamp API obejct.